python/triton/experimental/gluon/language/amd/cdna4/__init__.py

```diff
@@ -1,3 +1,40 @@
+from triton.experimental.gluon.language import _core as ttgl
+from ..._core import builtin, float32
+from ..._layouts import DotOperandLayout
+from .._layouts import AMDMFMALayout
 from ..cdna3 import buffer_load_to_shared, buffer_load, buffer_store
 
-__all__ = ["buffer_load_to_shared", "buffer_load", "buffer_store"]
+__all__ = ["buffer_load_to_shared", "buffer_load", "buffer_store", "mfma_scaled"]
+
+
+@builtin
+def mfma_scaled(lhs, lhs_scale, lhs_format, rhs, rhs_scale, rhs_format, acc, _semantic=None):
+    """
+    AMD MFMA scaled operation, supported only on CDNA4 hardware. This is thin
```

Comment:
Can you give a math description of what this operation is doing? Something like 

```
D = scale_mxfp(lhs, lhs_scale) * scale_mxfp(rhs, rhs_scale) + acc
```
where `scale_mxfp` applies the given E8M0 `*_scale` tensor to the input MXFP `lhs`/`rhs` tensor according to OCP microscaling format spec at https://www.opencompute.org/documents/ocp-microscaling-formats-mx-v1-0-spec-final-pdf.

Reply:
just updated. let me know whether it looks ok